torch/quantization/fuse_modules.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import copy
+
+import torch.nn._intrinsic.modules.fused as torch_fused
+
+def fuse_conv_bn(conv, bn):
```

Comment:
I feel I commented on one of the previous PRs, not sure which one. This is pretty general purpose - why it doesn't live in torch.nn.utils?

Reply:
The fusion code for inference is general. The part for training is really needed only for quantization aware training. Is it ok to move this to nn.utils?

Reply:
I don't mind either way. I guess, I'll move it to torch.nn.utils

Reply:
I am keeping the tests in the test_quantization, as there is no dedicated utils test file. LMK if I need to move it

Reply:
@dzhulgakov we had a discussion on the quantization working group, this is not going to cover all the cases, cc @ZolotukhinM

Reply:
Tried moving it to the `nn.utils` -- breaks everything. @jerryzh168 mentioned, might be due to circular dependency. Can move it if we converge to that decision

Reply:
Moved a part of it :)